app/views/pages/home.html.erb

```diff
@@ -7,7 +7,7 @@
   <div class="container">
     <h1>Demonic Cats</h1>
     <p>And the people that love them.</p>
-    <%= link_to "Create Article", '/article/new' %>
+    <%= link_to "Create Article", article_new_path %>
   </div>
 </div>
 </body>
```

Comment:
Missing new line.

A suggestion to prevent this from happening is using `git add -p`. You will be presented with each chunk of code to be committed and prompted to add it or not, but if you will see red blocks if there are any trailing whitespaces or missing new lines at the end. In this case you can choose to not to add that chunk, fix it and try it again later.

Reply:
That sounds very useful. I will try to incorporate it into my workflow.